queue_job/jobrunner/runner.py

```diff
@@ -22,13 +22,33 @@
 How to use it?
 --------------
 
-* Set the following environment variables:
+* Optionally adjust your configuration through environment variables:
 
-  - ``ODOO_CONNECTOR_CHANNELS=root:4`` (or any other channels configuration)
-  - optional if ``xmlrpc_port`` is not set: ``ODOO_CONNECTOR_PORT=8069``
+  - set ``ODOO_CONNECTOR_CHANNELS=root:4`` (or any other channels
+    configuration) if you don't want the default ``root:1``.
+
+  - if ``xmlrpc-port`` is not set, you can set it for the jobrunner only with:
+    ``ODOO_CONNECTOR_PORT=8069``.
+
+* Alternatively, configure the channels through the Odoo configuration
+  file, like:
+
+.. code-block:: ini
+
+  [options-queue_job]
```

Comment:
So, I originally made the section be called `[options-connector]` because I thought `anybox.recipe.odoo` could only make sections that were called `[options-something]`.

Today, while pondering your question, I decided to see how hard it would be to change `anybox` to create any kind of section name, and then I realised I was mistaken, and actually it's possible to create any section name.

So I just rewrote the section name to simply `[queue_job]`, while keeping backward compatibility with the old `connector` section name.